src/main/java/service/impl/FruitServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package service.impl;
+
+import java.util.Map;
+import model.FruitTransaction;
+import service.FruitService;
+import strategy.OperationHandler;
+
+public class FruitServiceImpl implements FruitService {

```

Comment:
The class implementation should not start with an empty line. Please remove the empty line at the beginning of the class definition.